### DISCUSSION TITLE
Progress Update Summary

### DISCUSSION THREAD

**Discussion by dgkeyes** [rfortherestofus/api-food-asset-map#43] [General]
Just making this as a way to keep a summary of progress, as much for myself as anything else. 

On **data import**, we're still working on a few things:

1. Getting ethnic markets coded correctly (client is doing this manually for us).
2. Making a separate accepts SNAP/WIC variable and applying it to all stores
3. Figuring out what category all stores that accept SNAP/WIC go into 

Once we have everything fixed above, we'll do reverse geocoding to get addresses for all places (or, as many as possible). 

On **building the map**, we have a basic prototype. We're still working on a few things:

1. Separating out the neighborhood demographics filter from the food resource category filters
2. Selecting icons for the food resources
3. Styling the tooltips

We'll still need to make static maps (see #41). I'll also need to do the styling for the entire website.

**Comment by dgkeyes** [comment]
Hi @katiejolly and @grahamammal, just wanted to send a quick update on this project. The client has asked if we can try to get this ready by May 24, which means this week is the final push. Here is my summary of remaining tasks. Please let me know if I've missed anything. 

**Importing Data**

We're close here!

We need to work on the final joining of SNAP/WIC stores with our full dataset, making a new variable for accepts SNAP/WIC. I can see in #26 that we need to finish doing what we can do programatically before creating a Google Sheet to get the client to help us with manual matching. This is the top priority right now because we need to get input from the client, which may take a couple days. If we can get this to them tomorrow (Monday), that would be ideal. @grahamammal if you can be the point person here, that would be great. 

Once we have the SNAP/WIC stuff taken care of, we then need to make sure the encoding works (#27) and do reverse geocoding (#25). 

**Building Map**

The biggest lift here is coming up with a way to separate out the filter that shows/hides demographic data (as discussed in #36). One idea I have here is, if we can't do this with leaflet alone, we could try using [crosstalk](https://rstudio.github.io/crosstalk/widgets.html). [Here is a nice example of this in practice](https://matt-dray.github.io/earl18-crosstalk/04_leaflet-flexdash-dt-crosstalk.html). @katiejolly if you can make this the first thing you work on, that would be great. 

We also need to work on a few styling things related the map, all of which are laid out in the [Build Map milestone](https://github.com/rfortherestofus/api-food-asset-map/milestone/2). 

**Deploy Map**

I've set the map up to deploy using Netlify (see the final result [here](https://apifoodmap.netlify.app/)). I'll work with the client to use a better custom domain and embed it on their site. 

Thanks to both of you for your hard work on this project! Please let me know if you need my support, especially with regard to the top issues mentioned above.

**Reply by katiejolly** [reply]
Sounds good! Planning to spend a big chunk of time on this today.

**Reply by dgkeyes** [reply]
Super, thanks!

**Reply by katiejolly** [reply]
For importing data, one thing I've noticed today is that many of the international grocery stores are duplicated. I'm not sure where this was introduced, but we should go through and edit this. I can probably do this tomorrow, but @grahamammal if you have a chance before I do, it would be very helpful!

**Reply by grahamammal** [reply]
Sounds good! I managed to get matches for about 200 of the 400 snap/wic stores. I would guess about 25% of the remaining have matches snap/wic stores, but it's not worth it to do the rest programmatically. Do you have an idea of the spreadsheet form that would be the best? I could list each snap/wic location, as well as the nearest location that we've collected otherwise, and ask them to check if they're a match? I might be able to list the closest 3 or so, but I'll need to check on that. Also, we'll need them to give a category to the snap/wic locations that don't have a match in the dataset

**Reply by dgkeyes** [reply]
Your approach sounds good @grahamammal. Do you mind just pushing it to a new sheet on the existing Google Sheet and letting me know when it's there. I'll check in then, make sure everything is clear, and then let the client know what they need to do. Thanks!

**Reply by grahamammal** [reply]
@katiejolly I noticed that too, I think many of the international grocery stores pulled from osm were duplicates of stores we had already pulled. I removed a couple of duplicates when I saw them earlier, but I can take another look at it

**Reply by grahamammal** [reply]
@katiejolly okay I took a closer look at the duplicates, looks like most of them are because they come both from the supermarkets dataset and the snap/wic dataset, so hopefully, after we finalize the join most of them will disappear
